### PR TITLE
Cache purges list

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: src/App/routes.jsx [hunk 1]
```diff
@@ -81,6 +81,12 @@ export default [
     ),
     path: '/secrets',
   },
+  {
+    component: loadable(() =>
+      import(/* webpackChunkName: 'Secrets' */ '../views/CachePurges')
```
Comment (eliperelman) [inline src/App/routes.jsx]: s/Secrets/CachePurges/

File: src/utils/constants.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ export const VIEW_WORKERS_PAGE_SIZE = 15;
 export const VIEW_CLIENTS_PAGE_SIZE = 20;
 export const VIEW_CLIENT_SCOPES_INSPECT_SIZE = 10;
 export const VIEW_SECRETS_PAGE_SIZE = 100;
+export const VIEW_CACHE_PURGES_PAGE_SIZE = 25;
```
Comment (eliperelman) [inline src/utils/constants.js]: Any reason 25 over using 20 like on the clients page?

File: src/views/CachePurges/index.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { hot } from 'react-hot-loader';
+import { Component } from 'react';
+import { Switch } from 'react-router-dom';
+import RouteWithProps from '../../components/RouteWithProps';
+import loadable from '../../utils/loadable';
+
+const ViewCachePurges = loadable(() =>
+  import(/* webpackChunkName: 'CachePurges.ViewCachePurges' */ './ViewCachePurges')
```
Comment (eliperelman) [inline src/views/CachePurges/index.jsx]: Do you view plural or singular?

File: src/views/CachePurges/index.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+);
+
+@hot(module)
+export default class PurgeCaches extends Component {
```
Comment (eliperelman) [inline src/views/CachePurges/index.jsx]: s/PurgeCaches/CachePurges/

File: src/views/CachePurges/index.jsx (hunk 3 above)
Comment (helfi92) [reply src/views/CachePurges/index.jsx]: @eliperelman Not sure I understand what you mean here. Would you say there is something to change here? ViewCachePurges contains a list of active cache purges.

File: src/App/routes.jsx (hunk 1 above)
Comment (eliperelman) [reply src/App/routes.jsx]: This is still outstanding.

Comment (helfi92) [reply src/App/routes.jsx]: My bad. Sending a fix now.

Comment (helfi92) [reply src/App/routes.jsx]: Done.

**Pull request merged**